src/views/clusteroverview/overview/components/details-card/hooks/useKubevirtCSVDetails.ts

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+
+import CatalogSourceModel from '@kubevirt-ui/kubevirt-api/console/models/CatalogSourceModel';
+import ClusterServiceVersionModel from '@kubevirt-ui/kubevirt-api/console/models/ClusterServiceVersionModel';
+import SubscriptionModel from '@kubevirt-ui/kubevirt-api/console/models/SubscriptionModel';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { K8sResourceCommon, useK8sWatchResources } from '@openshift-console/dynamic-plugin-sdk';
+import { useDeepCompareMemoize } from '@openshift-console/dynamic-plugin-sdk/lib/utils/k8s/hooks/useDeepCompareMemoize';
+
+import { CatalogSourceKind, ClusterServiceVersionKind, SubscriptionKind } from '../utils/types';
+
+import { useDebounceCallback } from './useDebounceCallback';
+
+const getKubevirtCSV = (clusterServiceVersions, installedCSV): ClusterServiceVersionKind => {
+  return clusterServiceVersions.find((csv) => csv?.metadata?.name === installedCSV);
+};
+
+const getKubevirtSubscription = (subscriptions) => {
+  return subscriptions.find((sub) => sub?.metadata?.name === 'hco-operatorhub');
+};
+
+const isPackageServer = (obj) =>
+  obj?.metadata?.name === 'packageserver' &&
+  obj?.metadata?.namespace === 'openshift-operator-lifecycle-manager';
+
+const catalogSourceForSubscription = (
```

Comment:
I think this function and those above in this file can be in a separate file, wdyt?

Reply:
I know we decided to keep utility functions out of component files to keep those leaner, but I didn't know that extended to hook files as well. I just looked at the hook files in `src/views/templates/list/hooks` and saw that the utility functions are included in the hook file there. 

If we're going to restrict having utility functions in hook files, then I think we need to make sure it's added to the conventions document and everyone knows so that we can be consistent. Personally, I think it makes sense to leave them there since hooks are about logic.